toto/runlib.py

```diff
@@ -100,6 +100,11 @@ def record_artifacts_as_dict(artifacts):
     return artifacts_dict
 
   for artifact in artifacts:
+
+    if not os.path.exists(artifact):
+      log.warning("path: {} does not exist, skipping..".format(artifact))
+      continue 
+
```

Comment:
My reasoning is that the message in case of error is different:
```
if islink:
  ....
elif isfile:
  ...
elif isdir:
  ...
else: #error
   log.error("This is either an unsupported file format or the file does not exist!")
```

Whereas the redundant conditions give you more granularity:
```python
if not os.path.exists(...):
   log.warning("COuld not find file...!")
...
if islink:
...
else:
   log.warning("This file type is not supported by in-toto!")
```